pages/deployment/aws.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
```

Comment:
Avoid defining a term by repeating the term itself. 

In this case, it is better to give a thumbnail sketch of the content, such as choosing instance type as memory optimized, selecting the OS, network considerations, etc.

Reply:
Good point.